openmrs/omod/src/main/java/org/projectbuendia/openmrs/web/controller/ProfileManager.java

```diff
@@ -33,14 +33,8 @@
 import java.io.InputStreamReader;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Date;
-import java.util.List;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
+import java.util.*;
```

Comment:
There's a setting in IntelliJ for this under Preferences > Code Style > Java on the "Imports" tab, named "Class count to use import with '*'".

The settings file at https://github.com/projectbuendia/buendia/blob/dev/intellij-idea-settings.jar should fix this too — it sets the class count to 99 so that `import` with `*` is basically never used.